js/typeFood.js

```diff
@@ -4,21 +4,24 @@ const searchInp = document.querySelector(".search-control");
 const holder = document.querySelector(".cards-food");
 let titelCat = document.querySelector(".header-search");
 let catName = localStorage.getItem('catFood');
+let parent = document.querySelector(".overlay");
```

Comment:
parent is null

Reply:
bc its functionality to close and open the popup

Reply:
yes, and you cant add style for null, so that makes an error
check it before replying,please!

Reply:
I already checked it, it works.